### PR TITLE
Add More button is working now

### PR DESCRIPTION
**Pull request by haridas**
Requires some more fixes to make it correct. I noted it on the comment.

### CODE REVIEW

File: static/js/custom_script.js [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
+    if (Element.size() <= 0 ) {
+        return; // No element to clone.
+    }
+    
+    var newElement = Element.clone(true);
+    
+    var total = $('#id_' + type + '-TOTAL_FORMS').val();
+    total++;
+    
+    newElement.find(':input').each(function() {
```
Comment (haridas) [inline static/js/custom_script.js:21]: Here the element ID increment need to be fixed. Right now the new formsets are being added on click. Also need to test duplicate addition of the forms since we are managing the form additions via javascript.

Need to test the POST request once the form creation is done.
